### DISCUSSION TITLE
I can't link a table row to a record on the table block

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#5391] [Help]
I have a tutorials CRUD app. The tutorials list page (with the table block) resides on the homepage. I've added new and detail screens.

When I link the row on the table block to `/tutorials/{{ Tutorials Table.Tutorials._id }}` it opens up the new row screen.

https://vimeo.com/698950130/283f26a1de

Any ideas about what's going on? I'm not sure if this is an issue or a lack of knowledge on my part.

**Comment by kasgithub8** [comment]
Hey @joebudi,
as far as I know and worked for me previously, it would be tutorial/:id which you can choose in the dropdown.
And because he handle the handlebars syntax, it redirects you to a different screen which is the normal outcome.
